### PR TITLE
Renamed subgroups of the connectedmachine extension and added a table format support

### PR DESCRIPTION
**Pull request by dorothyhu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedmachine

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to review? thanks

File: src/connectedmachine/azext_connectedmachine/generated/_help.py [hunk 1]
```diff
@@ -12,115 +12,110 @@
...
+--resource-group "myResourceGroup"
 """
 
-helps['connectedmachine machine-extension update'] = """
+helps['connectedmachine extension update'] = """
     type: command
     short-summary: "The operation to update the extension."
     examples:
-      - name: Update a Machine Extension
+      - name: Create or Update a Machine Extension (PATCH)
```
Comment (fengzhou-msft) [inline src/connectedmachine/azext_connectedmachine/generated/_help.py:87]: Remove `Create or `

File: src/connectedmachine/azext_connectedmachine/generated/_help.py [hunk 2]
```diff
@@ -12,115 +12,110 @@
...
 --resource-group "myResourceGroup"
 """
 
-helps['connectedmachine machine-extension create'] = """
+helps['connectedmachine extension create'] = """
     type: command
     short-summary: "The operation to create or update the extension."
     examples:
-      - name: Create or Update a Machine Extension
+      - name: Create or Update a Machine Extension (PUT)
```
Comment (fengzhou-msft) [inline src/connectedmachine/azext_connectedmachine/generated/_help.py:75]: Remove `or Update`

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**